modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,6 +216,55 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 ====
 
 
+==== Create databases using seeding options
+
+Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+To select a seed provider, you need to configure xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers].
+
+For more information on seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
+
+.Seed providers supported in Neo4j
+[cols="2,1,2",options="header"]
+|===
+| Seed provider
+| URL scheme
+| URI example
+
+| `FileSeedProvider`
+| `file:`
+| `file:/tmp/backup1.backup`
+
+| `URLConnectionSeedProvider`
+| `ftp:` +
+`http:` +
+`https:`
+| `ftp:://myftp.com/backups/backup1.backup` +
+`/http://myhttp.com/backups/backup1.backup` +
+`/https://myhttp.com/backups/backup1.backup`
+
+| `S3SeedProvider` label:deprecated[Deprecated in 5.26]
+| `s3:`
+| `s3://mybucket/backups/backup1.backup`
+
+| `CloudSeedProvider`
+| `s3:` +
+`gs:` +
+`azb:`
+| `s3://mybucket/backups/backup1.backup` +
```

Comment:
Interesting, there is a mismatch between the examples and this table here and on the other page. In the examples, the URLs are s3:/mybucket/backups/backup1.backup, gs:/mybucket/backups/backup1.backup, and azb:/mystorageaccount.blob/backupscontainer/backup1.backup, while in the tables we see double // after the s3, gs, and azb. I am not sure which one is correct.

Reply:
hmm, interesting indeed. If you look at the examples [here](https://neo4j.com/docs/operations-manual/current/backup-restore/restore-dump/#load-dump-cloud-storage), you'll we use double slashes after `s3:`, `gs:`, and `azb:`.

Reply:
Yeah, I don't know now. Maybe you can ask someone.